components/eamxx/src/physics/mam/eamxx_mam_dry_deposition_functions.hpp

```diff
@@ -0,0 +1,205 @@
+            team, fraction_landuse, atm.temperature, atm.pressure,
+            atm.interface_pressure, atm.hydrostatic_dp,
+            ekat::subview(qtracers, icol), dgncur_awet, wet_dens, obklen[icol],
+            surfric[icol], landfrac[icol], icefrac[icol], ocnfrac[icol],
+            friction_velocity[icol], aerodynamical_resistance[icol], dt,
+            // input-outputs
+            qqcw,
+            // outputs
+            ekat::subview(ptend_q, icol), ptend_lq,
+            ekat::subview(aerdepdrycw, icol), ekat::subview(aerdepdryis, icol),
+            // work arrays
+            rho, vlc_dry, vlc_trb, vlc_grv, dqdt_tmp);
+      });  // parallel_for for ncols
+}  // Compute_tendencies ends
+
+// Update interstitial aerosols using ptend_q tendencies
+void update_interstitial_mmrs(const MAMDryDep::view_3d ptend_q, const double dt,
+                              const int ncol, const int nlev,
+                              // output
+                              const mam_coupling::AerosolState dry_aero) {
+  const auto policy =
+      ekat::ExeSpaceUtils<MAMDryDep::KT::ExeSpace>::get_default_team_policy(
+          ncol, nlev);
+  static constexpr int nmodes = mam4::AeroConfig::num_modes();
+  Kokkos::parallel_for(
+      policy, KOKKOS_LAMBDA(const MAMDryDep::KT::MemberType &team) {
+        const int icol = team.league_rank();
+        Kokkos::parallel_for(Kokkos::TeamVectorRange(team, nlev), [&](int kk) {
+          for(int m = 0; m < nmodes; ++m) {
+            dry_aero.int_aero_nmr[m](icol, kk) +=
+                ptend_q(icol, kk, mam4::ConvProc::numptrcw_amode(m)) * dt;
+            for(int a = 0; a < mam4::AeroConfig::num_aerosol_ids(); ++a)
+              if(-1 < mam4::ConvProc::lmassptrcw_amode(a, m))
+                dry_aero.int_aero_mmr[m][a](icol, kk) +=
+                    ptend_q(icol, kk, mam4::ConvProc::lmassptrcw_amode(a, m)) *
+                    dt;
+          }
+        });  // parallel_for nlevs
+      });    // parallel_for icol
+}  // Update interstitial aerosols ends
+
+// Update cloud borne aerosols using qqcw
+void update_cloudborne_mmrs(const MAMDryDep::view_3d qqcw, const double dt,
+                            const int nlev_,
+                            // output
+                            const mam_coupling::AerosolState dry_aero) {
+  for(int m = 0; m < mam_coupling::num_aero_modes(); ++m) {
+    Kokkos::deep_copy(dry_aero.cld_aero_nmr[m],
+                      ekat::subview(qqcw, mam4::ConvProc::numptrcw_amode(m)));
+    for(int a = 0; a < mam_coupling::num_aero_species(); ++a) {
+      if(dry_aero.cld_aero_mmr[m][a].data()) {
+        Kokkos::deep_copy(
+            dry_aero.cld_aero_mmr[m][a],
+            ekat::subview(qqcw, mam4::ConvProc::lmassptrcw_amode(a, m)));
+      }
+    }
+  }
+}  // Update cloud borne aerosols ends
+
+// FIXME: remove the following function after implementing file read for landuse
+void populated_fraction_landuse(MAMDryDep::view_2d flu, const int ncol) {
```

Comment:
What is the relationship between this and the following (mam4xx/drydep.hpp) code? 

https://github.com/eagles-project/mam4xx/blob/2eee1988b1e6488c904e0a28564bdcadfa190d34/src/mam4xx/drydep.hpp#L32C10-L32C26

Reply:
Variable `fraction_landuse[n_land_type]` at https://github.com/eagles-project/mam4xx/blob/2eee1988b1e6488c904e0a28564bdcadfa190d34/src/mam4xx/drydep.hpp#L32C10-L32C26  is only used if we invoke ` DryDeposition::compute_tendencies` (see [here](https://github.com/eagles-project/mam4xx/blob/2eee1988b1e6488c904e0a28564bdcadfa190d34/src/mam4xx/drydep.hpp#L1129C5-L1129C39)) function. This function is only used in MAM4xx validation tests. EAMxx never invokes this function.

The `populated_fraction_landuse` in the EAMxx dry deposition interface is a temporary function that specifies fractional land use values for now. We will replace it with a reader that will read the values of fractional land use later.